rules/firewall_policies/hub_firewall.json

```diff
@@ -0,0 +1,69 @@
+{
+  "traffic-rule-collection-group": {
+    "priority": 400,
+    "network_rule_collections": {
+      "spoke-traffic": {
+        "action": "Allow",
+        "priority": 410,
+        "rules": {
+          "from-spoke": {
+            "protocols": [
+              "TCP"
+            ],
+            "source_addresses": [
+              "${spoke_vnet_address_pool}"
+            ],
+            "destination_addresses": [
+              "${hub_vnet_address_pool}"
+            ],
+            "destination_ports": [
+              "22"
+            ]
+          },
+          "to-spoke": {
+            "protocols": [
+              "TCP"
+            ],
+            "source_addresses": [
+              "${hub_vnet_address_pool}"
+            ],
+            "destination_addresses": [
+              "${spoke_vnet_address_pool}"
+            ],
+            "destination_ports": [
+              "22"
+            ]
+          },
+          "vnet-to-hub": {
+            "protocols": [
+              "TCP"
+            ],
+            "source_addresses": [
+              "${hub_gateway_address_prefixes}"
```

Comment:
why are the address while using the vpn gateway are not hub address? do you need this rule?